.github/workflows/test.yml

```diff
@@ -3,12 +3,13 @@ on:
   push:
     branches:
     - master
-    - feature/kfp
+    - feature/aip
+    - feature/aip
```

Comment:
Is it supposed to say "- feature/aip" twice?

Reply:
fixed, thanks!